.github/ISSUE_TEMPLATE/2-Bug_report.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Bug report
+about: You have found a bug
+labels: 'bug'
+---
+
+<!-- Please use the following issue template or your issue will be closed. -->
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Prerequisites
+
+## Expected Behavior
+
+<!--- Please describe the expected behavior -->
+
+## Current Behavior
+
+<!--- Please describe current behavior which needs to be fixed -->
+
+## Steps to Reproduce
+
+<!--- Please add steps to reproduce the bug -->
+
+1.
+2.
+3.
+4.
+
+## Possible Solution
+
+<!--- If you have ideas how to fix it, feel free to share these here -->
```

Comment:
<!--- If you have ideas how to fix it, feel free to share these here -->
=>
<!--- If you have ideas how to fix it, feel free to share them here -->